requirements.txt

```diff
@@ -0,0 +1,13 @@
+certifi==2020.12.5
```

Comment:
you should mention in the running section above that people can use this to pip install the modules they will need to run the script. some folks will be unfamiliar with python most likely

Reply:
This actually is there, let me add a separate subheading so it stands out a bit more.